src/public/css/landing.css

```diff
@@ -12,9 +12,10 @@ html, body{
     /* darkolivergreen might change to some url if professor provide it */
     background: #6d9d70 no-repeat 50% 50%;
     display: flex;
-    height: 40vh;
```

Comment:
this probably shouldn't be changed - why was it changed? I'd like to have the banner + our main "selling point" above the fold, or in other words show up on the screen on load without scrolling